server/team/team.contoller.js

```diff
@@ -2,20 +2,115 @@ const debug = require('debug');
 
 const Team = require('./team.model');
 const User = require('../user/user.model');
+const launchChromeAndRunLighthouse = require('../lighthouse/lighthouse');
 
 const log = debug('fcc:team:controller');
 
 function create(req, res) {
-  const { body: team } = req;
-  log(team);
-  return Team.create(team)
+  Team.create(req.body)
     .then((newTeam) => {
       if (!newTeam) {
         return res.status(500).json({ acknowladged: false });
       }
-      res.status(200).json({ acknowladged: true });
-      return newTeam._id;
+      return Team.findOneAndUpdate(
```

Comment:
I think you have got a bit delete happy here and merged `/create` with `/update`